lib/puppet/pops/lookup/lookup_key.rb

```diff
@@ -9,7 +9,7 @@ class LookupKey
   attr_reader :module_name, :root_key, :segments
 
   def initialize(key)
-    segments = split_key(key) { |problem| Puppet::DataBinding::LookupError.new("#{problem} in key: '#{key}'") }
+    segments = split_key(key) { |problem| Puppet::DataBinding::LookupError.new(_("%{problem} in key: '%{key}'") % { problem: problem, key: key }) }
```

Comment:
I left it because I'm sure there's a word for "hash key" in all of these languages. But I'm indifferent, we can skip it.